include/stdlib.h

```diff
@@ -81,6 +81,13 @@
 #define strtoll_l(s, e, b, l)  strtoll(s, e, b)
 #define strtoull_l(s, e, b, l) strtoull(s, e, b)
 
+/* Temporary fix for undefined exit() */
+
+#ifndef __KERNEL__
+#  undef  exit
+#  define exit nx_exit
```

Comment:
why

Reply:
CI error when building protected mode, exit is not defined in BUILD_PROTECTED, so this forwards exit to nx_exit.

This is temporary, and will be removed in the next PR.

Reply:
As lib_exit.c is not yet compiled I see no way around this issue. The user side of BUILD_PROTECTED needs a definition for exit(), and to me forwarding it to the nx_exit() call gate made sense. If you wish me to do it differently I can of course do that.

Reply:
But why not enable the compile of lib_exit?